nautobot/extras/models/customfields.py

```diff
@@ -187,7 +187,7 @@ def get_for_model(self, model):
 
 
 @extras_features("webhooks")
-class CustomField(BaseModel):
+class CustomField(BaseModel, ChangeLoggedModel):
```

Comment:
done to keep using the ViewTests that now are validating for ObjectChange

Reply:
I disagree with adding change-logging to a model just for the sake of making generic tests pass! I think I'd be OK with adding change-logging to CustomField *for functional reasons* but I don't think there's any value in adding it to RelationshipAssociation.

Reply:
I see your point, and also agree that adding just to make the test pass is the right approach but to customize the tests to match the expectations.
I proposed this just in case maybe these two models (that are sharing the same views), should be compliant as the rest of the models using them.
I will try your proposed way to customize them or a similar one to arrange it.